packages/suite/src/views/wallet/tokens/index.tsx

```diff
@@ -1,16 +1,22 @@
 import { useState } from 'react';
 
 import { WalletLayout } from 'src/components/wallet';
-import { useSelector } from 'src/hooks/suite';
+import { useDispatch, useSelector } from 'src/hooks/suite';
 import { CoinsTable } from './coins/CoinsTable';
 import { Route, Switch } from 'react-router-dom';
 import { TokensNavigation } from './TokensNavigation';
 import { HiddenTokensTable } from './hidden-tokens/HiddenTokensTable';
+import { goto } from 'src/actions/suite/routerActions';
 
 export const Tokens = () => {
     const [searchQuery, setSearchQuery] = useState('');
 
     const { selectedAccount } = useSelector(state => state.wallet);
+    const dispatch = useDispatch();
+
+    if (!selectedAccount.network?.features.includes('tokens')) {
```

Comment:
as the `network` can be undefined (you used `?`), it redirects user even when he refreshes in ETH tokens tab

You should use useEffect and check if account is properly loaded and then do decision 
~Another option is to get symbol from route hash and get features for this network and do it based on that. But I dont know if we do that and use it~

Reply:
Interesting, I couldn't replicate the error myself but I added the suggested solution, it will be safer like that anyway